GOVERNANCE.md

```diff
@@ -145,6 +143,11 @@ If and when contributors commitment levels change, contributors can consider ste
 
 Contact the Maintainers about changing to Emeritus status, or reducing your contributor level.
 
+### Emeritus contributors
+
+* [abonas](https://github.com/abonas)
+* [xeviknal](https://github.com/xeviknal)
+
```

Comment:
Minor:

We currently have 14 emeritus contributors (plus Alissa and Xavier who are being moved to this status). Perhaps, worth to mention all 16 emeritus contributors?

Reply:
+1 
Totally sense.
I was wondering if doing it in this PR or as a follow-up but I'll update this with the complete list.